edx/edx-ora2
# SE-2010 Implements the XBlock problem response report with a detailed assessment and responses report

Implements the `generate_report_data` method, allowing a custom report to be downloaded when using the Instructor Dashboard "Download Problem Response CSV" button.

The downloaded report contains the full list of enrolled learner submissions, and the assessments made against these submissions.

**JIRA tickets**: [OSPR-4073](https://openedx.atlassian.net/browse/OSPR-4073)

**Depencencies**: https://github.com/edx/edx-platform/pull/23109

**Merge deadline**: None

**Testing instructions**:

See https://github.com/edx/edx-platform/pull/23109 for sandbox and testing instructions.

[Sample report](https://github.com/edx/edx-ora2/files/5818640/OpenCraft_ORA2_2020_1_student_state_from_block-v1_OpenCraft%2BORA2%2B2020_1%2Btype%40chapter%2Bblock%4096abd57ebe08457cbc545a21f66b3240_2021-01-15-0525.csv.txt)

**Reviewers**
